src/__experimental__/components/grouped-character/grouped-character.spec.js

```diff
@@ -172,4 +173,16 @@ describe('GroupedCharacter', () => {
       });
     });
   });
+
+  it('the required prop is passed to the input', () => {
+    const wrapper = mountComponent({
+      label: 'required', required: true, separator, groups: basicGroupConfig, value: ''
+    });
+
+    const input = wrapper.find('input');
+    expect(input.prop('required')).toBe(true);
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.